python-checks/src/test/resources/checks/redundantJump.py

```diff
@@ -250,7 +251,7 @@ def redundant_return_inside_else_block():
         print(e)
     else:
         print(42)
-        return # Noncompliant
+        return # FN
```

Comment:
We should add tests for the cases where this PR removes FPs. I believe that's the same as the cases you added to `ControlFlowGraphTest`.